tests/x509/test_ocsp.py

```diff
@@ -753,3 +753,26 @@ def test_invalid_serialize_encoding(self):
             resp.public_bytes("invalid")
         with pytest.raises(ValueError):
             resp.public_bytes(serialization.Encoding.PEM)
+
+    @pytest.mark.supported(
+        only_if=lambda backend: (
+            backend._lib.CRYPTOGRAPHY_OPENSSL_110F_OR_GREATER),
+        skip_message="Requires OpenSSL 1.1.0f+",
+    )
+    def test_single_extensions(self, backend):
```

Comment:
Could you add one more test that asserts on the result of parsing this response when on OpenSSL < 1.1.0f?